storage/clickhousespanstore/reader.go

```diff
@@ -309,10 +309,10 @@ func (r *TraceReader) findTraceIDsInRange(ctx context.Context, params *spanstore
 	}
 
 	query += " AND -toUnixTimestamp(timestamp) <= -toUnixTimestamp(?)"
-	args = append(args, start.UTC().Format("2006-01-02T15:04:05"))
```

Comment:
Could you explain why the format change is needed?

Reply:
It's needed because search on clickhouse severs with non-UTC timezones works incorrectly. It happens because formatting  time the previous way deleted information about time zone, so search range "shifted" by differ between UTC and server time zone. Furthermore, passing time.Time as query parameter works perfectly fine.

Reply:
So line 312 adds the argument for the query on line 311 - `toUnixTimestamp(?)`. format `"2006-01-02T15:04:05"` does not encode timezone. Could we use a format that adds timezone? Then this is query is executed on clickhouse server.

I would prefer to be explicit and pass timestamp string instead of time instance.

Reply:
maybe we will also have to make sure the timestamp format on the writer side matches this format

Reply:
Writer side doesn't format time too, and I think that it's fine to relay on clickhouse-go driver on this.

Reply:
Clickhouse driver for database/sql supports passing time.Time as argument to statement.Exec(), and I passing it by formatting is a bit of wheel inventing.

Reply:
Alright, let's take this approach.